### DISCUSSION TITLE
Updating contrast turned screen purple

### DISCUSSION THREAD

**Discussion by govindrai** [MonitorControl/MonitorControl#1041] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

I adjusted contrast slider. after a couple of slides, the screen turned purple and is not reverting. 

### Steps to reproduce

1. Adjust contrast slider, many times (maybe 10-15 times)
2. Hope it doesn't turn purple :) 

### Expected behavior

It should not turn purple.

### Anything else?

![image](https://user-images.githubusercontent.com/13859249/168737593-cd1e91ed-7793-4928-9eb2-f13a727a335b.png)

Quitting monitor control instantly returned it back to its normal state. unfortunately I copied mac version number on top my clipboard and lost the purple monitor image. 

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.3.1 (21E258)
- Mac model: MacBook Pro (16-inch, 2021)
- MonitorControl version: 4.0.2
- Monitor(s): HP E243
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
The contrast slider sends an appropriate DDC command to the display - but contrast is not universally supported (or sometimes not properly supported) by some displays - so this is highly display firmware specific. If the display turned purple, the firmware might have messed up the color channels. If you find the OSD adjustment for this, you can fix it or you might have to reset the display.

**Comment by govindrai** [comment]
Makes sense. The HP E243 display is fairly new so I am surprised that it occurred. thanks for responding!
